src/build/helpers.ts

```diff
@@ -325,3 +325,21 @@ export function followTypeReferences(
     }
   }
 }
+
+export function assertUnique(list: string[]): string[] {
+  if (!Array.isArray(list)) return list;
```

Comment:
```suggestion
```

The type requirement `: string[]` is sufficient here (as this is just an internal function and thus is not exposed as a part of JavaScript library)